service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -24,10 +24,11 @@ class CaseData implements JsonSerializable
     #[Validator(NotEmpty::class)]
     public string $personType;
 
-    #[Validator(Regex::class, options: ["pattern" => "/^[0-9]{4}-[0-9]{2}-[0-9]{2}$/", "messages" => [
+    #[Annotation\Required(false)]
+    #[Validator(Regex::class, options: ["pattern" => "/^NA$|^[0-9]{4}-[0-9]{2}-[0-9]{2}$/", "messages" => [
```

Comment:
An empty value should be null or missing, not the string NA.
```suggestion
    #[Validator(Regex::class, options: ["pattern" => "/^[0-9]{4}-[0-9]{2}-[0-9]{2}$/", "messages" => [
```